core/app/c/[communitySlug]/forms/[formSlug]/edit/actions.ts

```diff
@@ -0,0 +1,37 @@
+"use server";
+
+import { logger } from "logger";
+
+import { db } from "~/kysely/database";
+import { isUniqueConstraintError } from "~/kysely/errors";
+import { getLoginData } from "~/lib/auth/loginData";
+import { autoRevalidate } from "~/lib/server/cache/autoRevalidate";
+import { defineServerAction } from "~/lib/server/defineServerAction";
+
+export const updateForm = defineServerAction(async function updateForm({ communityId, name }) {
+	const user = await getLoginData();
+
+	if (!user) {
+		return {
+			error: "Not logged in",
+		};
+	}
+	try {
+		const formUpdateQuery = await autoRevalidate(
+			db
+				.updateTable("forms")
+				.set({ name: name })
+				.where("communityId", "=", communityId)
+				.returning("id")
+		);
```

Comment:
This query will set every form in the community with id `communityId` to the same name. Instead, I think we should pass the form id to the server action and update the where clause to be `.where("id", "=", formid)`.

Reply:
that makes sense. I beleive i thought the prescence of a FKR would automatically update the the row at name_communityId